toolset/benchmark/test_types/verifications.py

```diff
@@ -41,58 +41,34 @@ def verify_headers(headers, url, should_be='json'):
     param `should_be` is a switch for the three acceptable content types
     '''
 
-    types = {
-        'json': 'application/json',
-        'html': 'text/html',
-        'plaintext': 'text/plain'
-    }
-    expected_type = types[should_be]
-
     problems = []
 
     for v in (v for v in ('Server', 'Date', 'Content-Type')
               if v.lower() not in headers):
-        problems.append(('warn', 'Required response header missing: %s' % v,
+        problems.append(('fail', 'Required response header missing: %s' % v,
                          url))
 
     if all(v.lower() not in headers
            for v in ('Content-Length', 'Transfer-Encoding')):
         problems.append((
-            'warn',
+            'fail',
             'Required response size header missing, please include either "Content-Length" or "Transfer-Encoding"',
             url))
 
-    content_type = headers.get('Content-Type', None)
+    content_type = headers.get('Content-Type')
```

Comment:
TODO(me): Figure out if I introduced an exception here when Content-Type is missing.